package.json

```diff
@@ -41,7 +41,7 @@
     "federalist": "npx gulp build",
     "lint": "npx gulp lintJS lintSass && npm run prettier:scss",
     "prestart": "npx gulp build",
-    "proof": "bundle exec htmlproofer --allow-hash-href=true --enforce-https=false --allow-missing-href=true --ignore-empty-alt=true --ignore-status-codes 0,403,429,503,302 --swap-urls 'https\\://designsystem.digital.gov/:/' --ignore-urls '/github.com/uswds/uswds/issues/' --ignore-files \"//select-a-language//,//2017//,//2018//,//2019//,//code-guidelines//\" ./_site",
```

Comment:
Thank you for the extra attention here! I had originally made this a part of the testing instructions to approve that this was indeed a change that brought potential value rather than a headache down the road.

> 2. Approve of the removal of error codes 503, 302. Would we want to keep these codes in the event one of the errors pop up in the future?
>    - 503 - This code indicates the server is not ready to handle the request
>    - 302 - This code indicates a specific URL has been moved temporarily to a new location
> 

Since these are temporary errors, how do you feel about adding them back to the ignore script?

Reply:
Since we both see value in keeping them ignored, I went ahead and restored 👍